### PR TITLE
Adds simulation finder tool figure 1 notebook to WIP

### PR DESCRIPTION
**Pull request by vicford**
This is a trimmed down version of Julia's variance_calculations notebook. AE navigation guide is also updated to incorporate this.

### CODE REVIEW

File: AE_navigation_guide.ipynb
Comment (claalmve) [inline AE_navigation_guide.ipynb:1]: Is there a reason for removing `threshold_event_types.ipynb` from the navigation guide?

File: work_in_progress/model_variance.ipynb [hunk 1]
```diff
@@ -0,0 +1,691 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fa85b245-a1f8-4603-ac51-472445e34292",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from climakitae.explore.uncertainty import weighted_temporal_mean\n",
```
Comment (claalmve) [inline work_in_progress/model_variance.ipynb:129]: Could remove the commented code and move the `weighted_temporal_mean` function to `utils` now that multiple files reference it.

File: AE_navigation_guide.ipynb
Comment (vicford) [reply AE_navigation_guide.ipynb:1]: This is a good catch - I'm not sure why it was deleted but adding back in

File: work_in_progress/model_variance.ipynb (hunk 1 above)
Comment (vicford) [reply work_in_progress/model_variance.ipynb:129]: It's already in utils, just forgot to remove this commented code out, good catch!

**Pull request merged**